lib/rubocop/path_util.rb

```diff
@@ -62,6 +62,13 @@ def self.reset_pwd
       @pwd = nil
     end
 
+    def self.chdir(dir, &block)
+      reset_pwd
+      Dir.chdir(dir, &block)
+    ensure
+      reset_pwd
+    end
```

Comment:
Test cases ware failed by caching `Dir.pwd` unfortunately.
So I added `RuboCop::PathUtil.chdir`, and use it instead of `Dir.chdir` to clear invalid cache.